tracks/tracks-bot-dev.py

```diff
@@ -0,0 +1,48 @@
+from dotenv import load_dotenv
+import os
+import logging
+from telegram import Update
+from telegram.ext import Application, CommandHandler, MessageHandler, filters, ContextTypes
+
+load_dotenv()
+TOKEN = os.getenv("TELEGRAM_BOT_TOKEN")
+
+
+# Configure logging
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO
+)
+logger = logging.getLogger(__name__)
+
+async def start_command(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Send a welcome message when the command /start is issued."""
+    await update.message.reply_text(
+        "Hi! I'm an echo bot. Send me any message and I'll repeat it back to you!"
+    )
+
+async def echo(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Echo the user message."""
+    await update.message.reply_text(update.message.text)
+
+async def error_handler(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Handle errors."""
+    await logger.error(f"Update {update} caused error {context.error}")
+
+def main() -> None:
```

Comment:
suggestion: rename `main()` to `start_bot()`

food for thought: do you think we should place the bot in its own class?